src/QTTrajectory.cc

```diff
@@ -74,11 +74,11 @@ std::pair<double,double> QTTrajectory::convertToVT(unsigned int which)
   fAntennaPos.setRThetaPhi(fAntennaRad, CLHEP::halfpi, fAngles[which]/360.0 * CLHEP::twopi);
   fAntennaNormal = (-fAntennaPos).unit(); // inward direction; to origin
 
-  // ask RHS object, acceleration as const reference
-  acc = pfieldmanager->GetChordFinder()->GetIntegrationDriver()->GetEquationOfMotion()->acc();
-
   // collect required values
-  G4double omega = pfieldmanager->GetChordFinder()->GetIntegrationDriver()->GetEquationOfMotion()->GetOmega().mag();
+  G4ThreeVector Bfield = pfieldmanager->GetChordFinder()->GetIntegrationDriver()->GetEquationOfMotion()->GetCachedFieldValue();
+  G4double omega = pfieldmanager->GetChordFinder()->GetIntegrationDriver()->GetEquationOfMotion()->CalcOmegaGivenB(Bfield).mag();
+  acc = pfieldmanager->GetChordFinder()->GetIntegrationDriver()->GetEquationOfMotion()->CalcAccGivenB(Bfield);
```

Comment:
@DSwinnockPhysics Could I just check exactly what acceleration should be returned here? Is it just the Lorentz force, or Lorentz+radiation reaction?

Reply:
Based on the code I have (which uses Seb's code), it's using Lorentz+radiation reaction.